Dockerfile

```diff
@@ -1,18 +1,26 @@
-# Take advantage of a multi-stage build to avoid dev-dependencies in the final image
-FROM node:16-alpine AS builder
+FROM node:16-alpine AS build
+ENV NODE_ENV=development
 
 WORKDIR /app
 
 COPY . .
 
-RUN yarn
+RUN yarn install
 RUN yarn build
 
-FROM node:16-alpine
+FROM node:16-slim AS dist
+ENV NODE_ENV=production
 
-WORKDIR /app
+WORKDIR /usr/local/share/graphql-inspector
+
+COPY --from=build /app/packages ./packages
+COPY --from=build /app/package.json ./
 
-COPY --from=builder /app/packages /usr/local/share/graphql-inspector
+RUN yarn install \
+    && yarn cache clean
```

Comment:
The cause of the error reported in #2141 is that the final Docker image did not have required `node_modules` installed.